jenkins/image_building/build-image.sh

```diff
@@ -3,15 +3,21 @@
 set -eux
 
 current_dir="$(dirname "$(readlink -f "${0}")")"
+REPO_ROOT="$(realpath "${current_dir}/../..")"
 
 cleanup() {
   deactivate || true
+  cd "${REPO_ROOT}" || true
```

Comment:
No need to use `cd` here.
  sudo rm -f "${REPO_ROOT}/${img_name}".{raw,qcow2}
  sudo rm -rf "${REPO_ROOT}/${img_name}.d"

Reply:
Good catch! Thanks